README.rst

```diff
@@ -74,6 +74,8 @@ Available corpora (version 1.1.0)
   11  Corpus of            cs      4100M    66428  12636867  stanza         CC-BY-SA
       Czech Verse                                            line
       (czverse)                                              word
+  12  Stichotheque         pt      11,8M     1702    168411  stanza         Unkwown
```

Comment:
```suggestion
  12  Stichotheque         pt      11.8M     1702    168411  stanza           12  Stichotheque         pt      11,8M     1702    168411  stanza         Unkwown
```
Is it really `Unkwown`?

Reply:
They wrote this:

> We do not have a formal license yet, but if you would like to use the files provided by Stichotheque, these are the informal terms:
> - You can use the files for research and education, but not for commercial purposes.
> - If you write an article that uses these files in some way, we ask you to cite one of our publications.

So I don't know which license to choose...

Reply:
That sounds a lot like CC-BY or an edu license. Let's keep it as Unknown. We need add links to their source projects at some point.

Reply:
Wich is the best option?

Reply:
> Wich is the best option?

Sorry I wrote this without refreshing the page. OK, Unknown.

Reply:
Please, it's `11.8M` not `11,8M`

Reply:
My fault, done!